src/lib/api/builder.ts

```diff
@@ -219,7 +219,9 @@ function fromRemoteProject(remoteProject: RemoteProject): Project {
     id: remoteProject.id,
     title: remoteProject.title,
     description: remoteProject.description,
-    thumbnail: `${BUILDER_SERVER_URL}/projects/${remoteProject.id}/media/thumbnail.png`,
+    thumbnail: remoteProject.is_template
```

Comment:
This change is to load the template thumbnail image from the public folder. 

same for video.